addOns/zest/src/main/java/org/zaproxy/zap/extension/zest/ExtensionZest.java

```diff
@@ -329,7 +335,7 @@ public void unload() {
                 }
             }
         }
-
+        this.removeClientHelper();
```

Comment:
I'd revert this, it's not really needed here (the one added in `ExtensionClientZest.unload` is enough).